tests/check/signed_video_helpers.h

```diff
@@ -32,13 +32,21 @@
 #define MANUFACT "manufacturer"
 #define ADDR "address"
 
+typedef enum { SV_RECURRENCE_DEFAULT = 1, SV_RECURRENCE_THREE = 3 } SignedVideoRecurrence;
+typedef enum {
+  SV_RECURRENCE_OFFSET_DEFAULT = 0,
```

Comment:
Is it possible to use RECURRENCE_OFFSET_DEFAULT instead of 0 to avoid bugs in case of change